Sources/Utility/ArgumentParser.swift

```diff
@@ -317,6 +317,9 @@ public final class ArgumentParser: ArgumentParserProtocol {
     /// Overview text of this parser.
     let overview: String
 
+    /// True if this parser is a subparser.
+    let isSubparser: Bool
```

Comment:
This is a bit unfortunate since it moves us further from having immutable command definitions that can be used as either a command or subcommand based on context, but that's admittedly an edge case.